application-admintools-test/application-admintools-test-docker/src/test/it/com/xwiki/admintools/test/ui/AdminToolsIT.java

```diff
@@ -0,0 +1,300 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.admintools.test.ui;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.panels.test.po.ApplicationsPanel;
+import org.xwiki.test.docker.junit5.TestConfiguration;
+import org.xwiki.test.docker.junit5.UITest;
+import org.xwiki.test.ui.TestUtils;
+import org.xwiki.test.ui.po.ViewPage;
+
+import com.xwiki.admintools.test.po.AdminToolsHomePage;
+import com.xwiki.admintools.test.po.AdminToolsViewPage;
+import com.xwiki.admintools.test.po.DownloadArchiveModalView;
+import com.xwiki.admintools.test.po.LastNLinesModalView;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@UITest
+class AdminToolsIT
+{
+    private static final List<String> supportedServers = List.of("TOMCAT");
+
+    private static final String USER_NAME = "JonSnow";
+
+    private static final DocumentReference ADMINTOOLS_CONFIGURATION_REFERENCE =
+        new DocumentReference("xwiki", Arrays.asList("AdminTools", "Code"), "Configuration");
+
+    private static final String ADMINTOOLS_CONFIGURATION_CLASSNAME = "AdminTools.Code.ConfigurationClass";
+
+    private static final String BACKEND_SECTION_VIEW_LAST_LOGS_MODAL_ID = "#configurationViewLastNLinesModal";
+
+    private static final String DOWNLOAD_FILES_MODAL_ID = "#downloadFilesModal";
```

Comment:
This doesn't belong here in the test. It should be in a page object. Tests shouldn't use (directly) any selectors normally.